lib/uart_blob/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources_ifdef(CONFIG_UART_BLOB_TX uart_blob_tx.c)
+zephyr_library_sources_ifdef(CONFIG_UART_BLOB_RX uart_blob_rx.c)
+zephyr_include_directories(.)
```

Comment:
Is this the way its typically done, or should the headerfile be moved to `nrf/include/...` ? Just check how the other libs does it

Reply:
The other libs don't seem to be very consistent, but there are examples of this being done in CMakeLists.txt (e.g. lib/multithreading_lock). I can make sure that cobs.h is not needed outside the directory and then remove this line.